README.md

```diff
@@ -70,3 +70,13 @@ If you want to run the unit tests:
 ```console
 meson test -C <build_dir>
 ```
+
+To run a binary without installing it:
+```console
+./<build_dir>/<subproject>/<binary>
+```
```

Comment:
Isn't this supposed to be `./<build_dir>/subprojects/<subproject>/<binary>`?

Reply:
Yeah 🤦